scripts/test-aws-creds.md

```diff
@@ -1,82 +0,0 @@
-### Introduction
```

Comment:
new place for this file?

Reply:
I will rewrite parts of this file and bring it back into the directory. Good call out